api/src/wfl/sink.clj

```diff
@@ -242,21 +243,104 @@
                                 :fromOutputs fromOutputs}))))
     (assoc request :dataset dataset')))
 
+(defn ^:private push-job [{:keys [details] :as _sink} job]
+  (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
+    (jdbc/insert! tx details job)))
+
+(defn ^:private peek-job-queue [{:keys [details] :as _sink}]
+  (let [query "SELECT * FROM %s
+               WHERE status = 'succeeded' AND consumed IS NULL
```

Comment:
Double checking that succeeded is meant to be in lowercase.

Reply:
yeah - tdr job statuses are all lower case. Probably should to a `to-lower` to make sure.